src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -95,104 +95,44 @@
 """
 
 
-class FlaxBertLayerNorm(nn.Module):
```

Comment:
Let's use the official `flax.linen.LayerNorm` module to reduce future maintenance